### ISSUE TITLE
Fix Qiskit deprecation warnings

### ISSUE THREAD

**Issue by natestemen** [unitaryfund/mitiq#1533]
Seeing the following warnings on CI:
```
mitiq/interface/mitiq_qiskit/tests/test_qiskit_utils.py::test_execute
mitiq/pec/representations/tests/test_learning.py::test_biased_noise_loss_function_qiskit[operations0]
  /Users/runner/work/mitiq/mitiq/mitiq/interface/mitiq_qiskit/qiskit_utils.py:124: PendingDeprecationWarning: The qiskit.Aer entry point will be deprecated in a future release and subsequently removed. Instead you should use this directly from the root of the qiskit-aer package.
    backend=qiskit.Aer.get_backend("aer_simulator_density_matrix"),

mitiq/interface/mitiq_qiskit/tests/test_qiskit_utils.py::test_sample_bitstrings_with_backend
mitiq/interface/mitiq_qiskit/tests/test_qiskit_utils.py::test_compute_expectation_value_on_noisy_backend_with_qiskit_backend
  /Users/runner/hostedtoolcache/Python/3.9.14/x64/lib/python3.9/site-packages/qiskit/providers/fake_provider/fake_backend.py:462: PendingDeprecationWarning: "warnings" argument will be deprecated as part of the qiskit-aer 0.12.0 and subsequently removed
    noise_model = NoiseModel.from_backend(self, warnings=False)

mitiq/zne/tests/test_zne.py::test_qiskit_execute_with_zne
  /Users/runner/work/mitiq/mitiq/mitiq/interface/mitiq_qiskit/qiskit_utils.py:176: PendingDeprecationWarning: The qiskit.Aer entry point will be deprecated in a future release and subsequently removed. Instead you should use this directly from the root of the qiskit-aer package.
    backend=qiskit.Aer.get_backend("aer_simulator"),
```

First and last issues want us to directly use the [`qiskit-aer`](https://pypi.org/project/qiskit-aer/) package, and the second should be fixed by removing the `warnings` key word argument.

**Comment by andre-a-alves** [comment]
I can fix this issue when I fix #1835

**Comment by natestemen** [comment]
@andre-a-alves it looks like this was resolved by #1847, right?

**Comment by andre-a-alves** [comment]
> @andre-a-alves it looks like this was resolved by #1847, right?

Yes, it was
